src/main/java/racingcar/model/Car.java

```diff
@@ -0,0 +1,20 @@
+package racingcar.model;
+
+public class Car {
+    private final int carNumber;
+    private int carMovingDistance;
```

Comment:
```suggestion
    private int movingDistance;
```

개인적으로 Car 객체 상태값은 car prefix 부분을 제거해도 된다고 생각해요

- `car.자동차이름`
- `car.자동차생성시간`
보다는 

중복을 제거한 아래 표현이 더 자연스럽지 않을까요?
- `car.이름`
- `car.생성시간`